symbolic-debuginfo/src/macho/bcsymbolmap.rs

```diff
@@ -187,6 +191,194 @@ impl<'a, 'd> Iterator for BcSymbolMapIterator<'a, 'd> {
 
 impl FusedIterator for BcSymbolMapIterator<'_, '_> {}
 
+#[derive(Debug, Error)]
+#[error("{kind}")]
+pub struct PListError {
```

Comment:
This is called `PListError` but is used for parsing a `BitcodeUuidMapping`. Is this intentional?

Also, this error would benefit from a doc comment.

Reply:
Yes, kind of.  It's only returned from the `.parse_plist*` methods and the error kinds are primarily dealing with plists.  Making it a `BitcodeUuidMappingError` and `BitcodeUuidMappingErrorKind` was also just too wieldy plus it is weird to jam plist errors in such a generic name.  Unless we make a `BitcodeUuidMappingError` which only provides a source but does not try to provide a kind.  Which could also be an option.

Reply:
Then let's call it a `DSymPlist` instead of a `BitcodeUuidMapping`, and the associated error `DSymPlistError`.